models/Patient.js

```diff
@@ -1,26 +1,73 @@
 const mongoose = require('mongoose');
 
-const PatientSchema = new mongoose.Schema({ //patient schema
-    name: {
+const PatientSchema = new mongoose.Schema({
+  //patient schema
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  age: {
+    type: Number,
+    required: true,
+  },
+  underlying: {
+    type: Boolean,
+    required: true,
+  },
+  date: {
+    type: Date,
+    default: Date.now,
+  },
+  patientEntry: [
+    {
+      date: {
+        type: Date,
+        default: Date.now,
+        required: true,
+        unique: true,
+      },
+      symptom1: {
         type: String,
-        required: true
-    },
-    email: {
+      },
+      symptom2: {
         type: String,
-        required: true,
-        unique: true
-    },
-    date: {
-        type: Date,
-        default: Date.now
+      },
+      symptom3: {
+        type: String,
+        // required: true,
+      },
+      symptom4: {
+        type: String,
+        // required: true,
+      },
+      temp: {
+        type: String,
+        // required: true,
+      },
+      comment: {
+        type: String,
+      },
+      doctorNote: {
+        type: String,
+      },
+      immediateAttention: {
+        type: Boolean,
+      },
```

Comment:
We did not agree on having the same schema for patients' information and patient entries. Doesn't it change many routes and what we have done already? Also isn't this Yergu's job make it work?